docs/ContributingCode.md

```diff
@@ -68,11 +68,18 @@ There are 3 types of static analysis that are enforced on every PR:
 ## Publishing a Release
 Only repo maintainers can publish a release. The process is kicked off by pushing a tag in the format `v0.7`. We follow [Semantic Versioning](https://semver.org/) when deciding on the next version number.
 
-This will trigger an Actions workflow that results in a new release being published. It performs these steps:
-- Validates that the SHA referenced by the tag is in the history of the `main` branch
-- Runs `publish.ps1` to build self-contained binaries for each platform. This script also embeds the version number (extracted from the tag) into each binary.
-- Creates a release in this repo with the self-contained binaries, uses the contents of `RELEASENOTES.md` as the release description.
-- Moves the contents of `RELEASENOTES.md` to a version specific file under the releasenotes folder, then empties out `RELEASENOTES.md` and commits the 2 files
+### How to publish a release
+
+1. Check [`RELEASENOTES.md`](http://releasenotes.md/) to see if there is anything to release.
+2. Switch to the `main` branch and `pull` the latest.
+3. *optional* List the tags —> `git tag`
+4. Tag the version —> `git tag v5.0`
+5. Push the tags up —> `git push --tags`
```

Comment:
not sure if it's super clear that by `->` we mean e.g. so I would just add an `e.g.` after `->`:

```
Tag the version -> e.g. `git tag v5.0`
```